### ISSUE TITLE
Make it possible to require marked by making a separate cjs release branch

### ISSUE THREAD

**Issue by Jelmerro** [markedjs/marked#2931]
**What pain point are you perceiving?.**
Marked stop working in CommonJS environments that also use webpack, as it's tree-shaken away entirely due to switching to modules. I am working on an Electron app that uses marked, so I am stuck using CommonJS, but since V6 marked is not included when I make a build with webpack. I tried doing an `async import` and while that mostly worked (except that types are a pain now), it is tree-shaken when build with webpack.

**Describe the solution you'd like**
Release a version that can be required as before under an npm tag like `cjs`, since importing in CommonJS can work for modules, but even if it does it's not going to be included in the webpack build. Or let me know a way that I can prevent this from happening to marked, though I have not found one yet.

In short, it would greatly help if there was an npm tag that points to the latest stable CommonJS supported version.
